infrastructure/web/lesson_video_buckets.tf

```diff
@@ -0,0 +1,23 @@
+// I want to create a bucket in gcp to hold lesson videos
+resource "google_storage_bucket" "lesson_videos" {
+  name          = "lesson-videos-${var.environment}"
+  location      = var.region
+  force_destroy = var.environment != "production"
+
+  versioning {
+    enabled = false
+  }
+
+  cors {
+    origin          = [var.retool_origin]
+    method          = ["GET", "HEAD", "PUT", "POST", "DELETE"]
+    response_header = ["Content-Type", "x-goog-acl"]
```

Comment:
As I read it the response header is only relevant for PUT/POST requests so if this is read only I don't think we need this.

Reply:
[see comment](https://github.com/oaknational/Oak-Web-Application/pull/1669#discussion_r1230688769)